tests/interfaces/test_qnode_jax.py

```diff
@@ -137,7 +137,7 @@ def test_adjoint_reuse_device_state(mocker):
     """Tests that the jax interface reuses the device state for adjoint differentiation"""
     dev = qml.device("default.qubit", wires=1)
 
-    @qml.qnode(dev, interface="jax", diff_method="adjoint")
+    @qnode(dev, interface="jax", diff_method="adjoint")
     def circ(x):
         qml.RX(x, wires=0)
         return qml.expval(qml.PauliZ(0))

```

Comment:
I assume all these tests that now run on the old qnode have equivalents for the new one?

Reply:
Yep! Inside the interfaces folder, there is `tests/test_batch_*.py`. These are the equivalent tests for the new interfaces.

I only keep the old ones for coverage for the time being.